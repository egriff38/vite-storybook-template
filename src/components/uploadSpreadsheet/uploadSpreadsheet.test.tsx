import { mount } from "@vue/test-utils";
import UploadSpreadsheet from "./uploadSpreadsheet.vue";

test("hello world", async () => {
  const wrapper = mount(UploadSpreadsheet);
  expect(wrapper.html()).toMatchSnapshot();
});