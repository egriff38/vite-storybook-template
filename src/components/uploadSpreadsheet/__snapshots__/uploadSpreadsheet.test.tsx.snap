// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello world 1`] = `
"<el-upload file-list=\\"\\" auto-upload=\\"false\\" multiple=\\"\\" drag=\\"\\" action=\\"\\"><i class=\\"el-icon-upload\\"></i>
  <div class=\\"el-upload__text\\"><em>click to upload</em></div>
</el-upload>"
`;
